package.json

```diff
@@ -64,6 +64,7 @@
     "eslint-config-prettier": "^6.6.0",
     "eslint-plugin-prettier": "^3.1.1",
     "eslint-plugin-react": "^7.17.0",
+    "node-sass": "^4.13.0",
```

Comment:
- updated `node-sass` from v4.12.0 to latest (no breaking changes: https://github.com/sass/node-sass/releases/tag/v4.13.0)
- made it a devDependency indead